src/Components/Address/useAddressAutocomplete.ts

```diff
@@ -23,13 +23,17 @@ export interface AddressAutocompleteSuggestion
   entries: number
 }
 
+interface ServiceAvailability {
+  loaded: boolean
+  enabled?: boolean
+}
```

Comment:
I made loaded/enabled two separate properties to not get loose with the meaning of 'enabled'.

Reply:
`enabled` + `ready`? I suppose there's not really any "loading" happening.